From 8b4749062c4ff4a998c1004cf52b862daf6ecc77 Mon Sep 17 00:00:00 2001
From: Rui Teng <rui.teng@amd.com>
Date: Thu, 26 Sep 2019 17:13:38 +0800
Subject: [PATCH 3889/4256] Revert "drm/amdgpu: enable full ras by default"

This reverts commit 5e6ebea39014345b35e1751fa1ca687e3315a2aa.

The original patch will cause kfdtest failed on mi50 and mi60.

Change-Id: Idef24a8dc6318a30b2c212f0c615c5ec8239d1a4
Signed-off-by: Rui Teng <rui.teng@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
index 355aa4221a88..952ed019fb22 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
@@ -155,7 +155,7 @@ struct amdgpu_mgpu_info mgpu_info = {
 	.mutex = __MUTEX_INITIALIZER(mgpu_info.mutex),
 };
 int amdgpu_ras_enable = -1;
-uint amdgpu_ras_mask = 0xffffffff;
+uint amdgpu_ras_mask = 0xfffffffb;
 
 /**
  * DOC: vramlimit (int)
-- 
2.17.1

